Fixtures/Templates/MakePackageWithTemplate/templates/new-package/ExeTemplate/.gitignore

```diff
@@ -0,0 +1,7 @@
+.DS_Store
+/.build
+/Packages
+/*.xcodeproj
+xcuserdata/
+DerivedData/
+.swiftpm/xcode/package.xcworkspace/contents.xcworkspacedata
```

Comment:
I think we can remove this?

Reply:
I think historically it has been there since a lot people use Xcode to work on packages, and Xcode tends to write contents to this file.  But it could probably be removed and people could add it as needed.  @neonichu what do you 
think?